install.md

```diff
@@ -154,7 +154,7 @@ For convenience, we also provide packages through several package managers. Many
 Binary conda packages are on [conda-forge][5] for Linux, macOS, and Windows
 for the following versions:
 
-* Python 3.6, 3.7, 3.8, 3.9
+* Python 3.6, 3.7, 3.8, 3.9, 3.10
 * R 3.6, 4.0
```

Comment:
@xhochy should the R versions also be updated? (Are there packages for 4.0 and 4.1 now?) Should we just say "current and past releases" to be future-proof?

Reply:
> @xhochy should the R versions also be updated? (Are there packages for 4.0 and 4.1 now?) Should we just say "current and past releases" to be future-proof?

It's possibly better to be explicit for people looking up what should be supported.

Reply:
Being clear about what has been tested is very helpful for newcomers. It makes starting use of Arrow much easier and will grow the community. Many of the R builds are for 3.6, probably some policy on which versions to support and for how long will be useful as this will guide updating the versions in CI.